### PR TITLE
Add workflow executor

### PR DESCRIPTION
**Pull request by Iqqdd99**

### CODE REVIEW

File: installation-scripts-onm/MASTER_START_SCRIPT.sh [hunk 1]
```diff
@@ -108,4 +108,16 @@ if [ "$SERVERLESS_ENABLED" == "yes" ]; then
 
   kubectl apply -f https://raw.githubusercontent.com/kubevela/samples/master/06.Knative_App/componentdefinition-knative-serving.yaml
   echo "Serverless installation completed."
+fi
+
+if [ "$WORKFLOW_ENABLED" == "yes" ]; then
+  echo "Workflow installation.";
+
+  helm install argo-workflows argo-workflows --repo https://argoproj.github.io/argo-helm --namespace argo --create-namespace --values workflow/workflow.yaml;
```
Comment (yoctozepto) [inline installation-scripts-onm/MASTER_START_SCRIPT.sh:116]: AFAIU, SAL does not see/transfer these files so your best bet for now is a workaround like above on L109.

**Review by yoctozepto — CHANGES_REQUESTED** [review]

File: installation-scripts-onm/MASTER_START_SCRIPT.sh (hunk 1 above)
Comment (Iqqdd99) [reply installation-scripts-onm/MASTER_START_SCRIPT.sh]: I've just added the wget file fetching like above.

Comment (yoctozepto) [reply installation-scripts-onm/MASTER_START_SCRIPT.sh]: Thanks, LGTM.

**Review by yoctozepto — APPROVED** [review]
Let's now wait for both @jmarchel7bulls and @ankicabarisic to have a look before merging.

**Comment by ankicabarisic** [comment]
@yoctozepto I am not aware of the workflow in this context. however, Do not see any change to which we object, so for us, it is fine. Please validate the impact of the changes by running cluster & app deployment over cloud/edge resources as usual.

**Comment by jmarchel7bulls** [comment]
The content looks good, but I’m concerned about merging directly to the main branch. I believe it would be more appropriate to merge into the dev branch first. Once the tests are passed, we can then merge it into the main branch. @yoctozepto please add your thoughts

**Comment by yoctozepto** [comment]
Good thinking @jmarchel7bulls

**Pull request merged**